portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,7 +14,9 @@
 
 package com.google.sps.servlets;
 import java.util.*;
+import com.google.gson.Gson;
```

Comment:
In the future, please be consistent with how you order the imports. (E.g. always sort)